core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala

```diff
@@ -449,6 +449,11 @@ class RichPresentationCompiler(
       && !sym.nameString.contains("$"))
   }
 
+  private def noDefinitionFound(tree: Tree) = {
+    logger.warn("No definition found. Please report to https://github.com/ensime/ensime-server/issues/492 with description of what did you expected. symbolAt for " + tree.getClass + ": " + tree)
+    Nil
```

Comment:
I've improved this method a little with adding more description what is happening